dev/prod/src/main.ts

```diff
@@ -36,13 +36,13 @@ setMetadata(meetingPlugin.metadata.ClientUrl, `${meetingHost}:${meetingPort}`)
 setMetadata(pluginCore.metadata.ClientUrl, clientUri)
 
 addEventListener('Token', async (event, data) => {
+  setMetadata(pluginCore.metadata.Token, data)
```

Comment:
What about AccountId? It is also required by `plugin-core`?

Reply:
I think it is not needed and is outdated, I could not find a place where it would be used.

Reply:
Ok, we need to remove it in this case.